src/ocean/core/BitArray.d

```diff
@@ -880,26 +832,30 @@ struct BitArray
 
 
     /**
-     * Updates the contents of this array with the result of a bitwise and
+     * Updates the contents of this array with the result of a bitwise `op`
      * operation between this array and the supplied array.
      *
+     * Supported are bitwise assignment "&amp;=", "|", "^"
```

Comment:
shoudn't it be "|=",  "^=" instead?

Reply:
In any case consistent

Reply:
Well the `op` parameter takes the value `|`, not `|`. But to show the difference with a non-assign version, I wrote "bitwise *assignment*"

Reply:
then you could remove the assignment from `&=` to keep it consistent as @alaksiej-stankievicx-sociomantic suggested

Reply:
Ah right, I didn't see that. Will fix.